MainApp/templates/MainApp/profile.html

```diff
@@ -75,21 +75,23 @@ <h3>{{profile_user.bio}}</h3>
          <span style="font-size: 20px; color: rgb(99, 99, 99);"> - {{post.pub_date}}</span>
          <div style="word-wrap: break-word; padding-top: 5px; line-height: 130%;">{{ post.visible_info | linebreaksbr | nbsp }}</div>
          <div class="fakeimg" style="height:60px;">Image</div>
-       </div>
+         </div>
        <!--If the post user equals the authenticated user, do not show like button-->
-       {% ifnotequal user.username post.tweet_creator.username %}
-       <form name="like_tweet" method="POST" action="{% url 'MainApp:homepage' %}">
-         {% csrf_token %}
-         <!--Show either like or unlike depending on whether or not user has liked the tweet-->
-         {% if not isLiked %}
-           <button name="like_button" value="{{user.username}},{{post.pk}}" class="like">Like</button>
-          {% else %}
-            <button name="unlike_button" value="{{user.username}},{{post.pk}}" class="like">Unlike</button>
-          {% endif %}
-        </form>
-        {% endifnotequal %}
```

Comment:
Remove the check to prevent the current authenticated user from liking their own tweet.